src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
+
+    margin-bottom: unit(45px / @base-font-size-px, em);
+
+    .respond-to-min(1024px, {
+        margin-top: unit(60px / @base-font-size-px, em);
+    });
+
+    .js-mobile-carousel_media .image-container {
+        margin-left: 10px;
+    }
+
+    .js-mobile-carousel_section {
+        margin-top: unit(30px / @base-font-size-px, em);
+    }
+
+    .js-mobile-carousel_section + .js-mobile-carousel_section {
+        .respond-to-min(600px, {
+            border-top: 1px solid @gray-50;
+            padding-top: unit(30px / @base-font-size-px, em);
+        });
+    }
+
+    .media_image__100 {
+        width: 100px;
+    }
+
+    .media_image__130-to-150 {
+        width: 130px;
+        .respond-to-min(800px, {
+            width: 150px;
+        });
+    }
+
+    // Slick carousel overrides https://github.com/kenwheeler/slick/.
+    &.slick-slider {
+          margin-top: (@grid_gutter-width * 2);
+          padding-top: (@grid_gutter-width * 2);
+          border-top: 1px solid @gray-50;
+          border-bottom: 1px solid @gray-50;
+          padding-bottom: (@grid_gutter-width / 2);
+
+        .slick-dots {
+            top: 30px;
+
+            li {
+                margin: 0;
+
+                button {
+                    &:before {
+                        color: @gray-50;
+                        font-size: 40px;
+                        opacity: 1;
+                    }
+
+                    &:hover:before {
+                        color: @gray;
+                    }
+                }
+
+            }
+
+            li.slick-active {
+                button:before {
```

Comment:
Gotcha, my mistake I misunderstood what you meant by nesting. Fixed in https://github.com/cfpb/cfgov-refresh/commit/761ecc8c125e9e3a944f4a8b6f0b10d034f2c28e